api4/system_test.go

```diff
@@ -242,7 +242,8 @@ func TestGetLogs(t *testing.T) {
 	Client := th.Client
 
 	for i := 0; i < 20; i++ {
-		mlog.Info(fmt.Sprint(i))
+		logMessage := fmt.Sprint(i)
+		mlog.Info(logMessage)
```

Comment:
It's is a bit artificial to create a tmp variable for this. What do you think about:
```suggestion
		mlog.Info(strconv.Itoa(i))
```

Reply:
Good point